lib/Red/ColumnMethods.pm6

```diff
@@ -62,6 +62,11 @@ method day($base where { .returns ~~ (Date|DateTime|Instant) }:) {
     Red::AST::DateTimePart.new(:$base, :part(Red::AST::DateTime::Part::day)) but Red::ColumnMethods
 }
 
+#| Return the date from a datetime, timestamp etc
+method date($base where { .returns ~~ (Date|DateTime|Instant) }:) {
```

Comment:
I was planning on calling it `.yyyy-mm-dd()`, and maybe also implement `.mm-dd-yyyy()` and `.dd-mm-yyyy()` (maybe `.hh-mm-ss()` for DateTime|Instant), what do you think?

Reply:
Fine with the name.

Reply:
Updated the name.  I've left the other variants because I think they need strftime or similar and that doesn't seem to work the same on Pg, as SQLite.